### PR TITLE
feat: Add link to ORBIT Blog on portal

### PR DESCRIPTION
**Pull request by jcbcapps**
# SC-642

## Proposed changes
Adds dropdown under About Us in the top navigation, for the About Us page as well as the ORBIT blog page.

---

## Reviewer notes
Hover over 'About Us' in the header navigation and you should see a dropdown menu appear:
- Click 'About the USSF' and verify that it takes you to `http://localhost:3000/about-us`
- Click 'ORBIT blog' and verify that it takes you to `http://localhost:3000/about-us/orbit-blog`

## Setup
In ussf-portal-client run `yarn services:up && yarn dev` and in ussf-portal-cms run `yarn dev`

---

### As the original developer, I have

- [ ] Met the acceptance criteria, or will meet them in subsequent PRs or stories
- [ ] Created new stories in Storybook if applicable
  - [ ] Checked that all Storybook accessibility checks are passing
- [ ] Created/modified automated unit tests in Jest
  - [ ] Including jest-axe checks when UI changes
- [ ] Created/modified automated E2E tests in Cypress
  - [ ] Including cypress-axe checks when UI changes
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)
- [ ] Requested a design review for user-facing changes
- For any new migrations/schema changes:
  - [ ] Followed guidelines for zero-downtime deploys

### As code reviewer(s), I have

- [x] Pulled this branch locally and tested it
- [x] Reviewed this code and left comments
- [x] Checked that all code is adequately covered by tests
- [x] Made it clear which comments need to be addressed before this work is merged
- [x] Considered marking this as accepted even if there are small changes needed
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [x] Checked responsiveness in mobile, tablet, and desktop
  - [x] Checked keyboard navigability
  - [x] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [x] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [x] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

### As a designer reviewer, I have

- [ ] Checked in the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

### As a test user, I have

- Run through the [Test Script](hhttps://docs.google.com/spreadsheets/d/1eV8UK0aJZ0qrzjnXf5SJ_uRiV7bpsj3yrhRFEkjOvV4/edit?usp=sharing):
  - [ ] On commercial internet in IE11
  - [ ] On commercial internet in Firefox
  - [ ] On commercial internet in Chrome
  - [ ] On commercial internet in Safari
  - [ ] On NIPR in IE11
  - [ ] On NIPR in Firefox
  - [ ] On NIPR in Chrome
  - [ ] On NIPR in Safari
  - [ ] On a mobile device in Firefox
  - [ ] On a mobile device in Chrome
  - [ ] On a mobile device in Safari
- [ ] Added any anomalous behavior to this PR

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -27,10 +30,50 @@ const Header = () => {
...
   }
 
-  const navItems = [
-    <NavLink key="nav_about" href="/about-us">
-      About us
+  const onToggle = (index: number): void => {
+    setIsOpen((prevIsOpen) => {
+      const newIsOpen = [false, false]
+
+      newIsOpen[index] = !prevIsOpen[index]
```
[the remark this answers is not in the document]
Comment (jcbcapps) [reply src/components/Header/Header.tsx:37]: I saw the warning but wasn't sure how to address it. But I agree, with no user input I think it makes sense to disable it.

File: src/components/Header/Header.module.scss [hunk 2]
```diff
@@ -66,6 +66,20 @@
...
+      font-weight: normal;
+    }
+
+    .usa-nav__primary button[aria-expanded='false']:hover {
+      background-image: url(/vendor/img/usa-icons-bg/expand_more--white.svg);
+    }
+
+    .usa-nav__primary
+      > .usa-nav__primary-item
+      > button.usa-accordion__button.usa-nav__link.usa-current::after {
```
[the remark this answers is not in the document]
Comment (jcbcapps) [reply src/components/Header/Header.module.scss:80]: I'm not seeing any warning in storybook for the Header component. Could you post the warning either here or in Slack?

[the remark this answers is not in the document]
Comment (gidjin) [reply src/components/Header/Header.module.scss:80]: This was due to my specific browser, I don't see this is Chrome or Safari.

**Review by gidjin — APPROVED** [review]
LGTM!

**Pull request merged**